### ISSUE TITLE
bug hunt 2!

### ISSUE THREAD

**Issue by therealjustinmiller79** [jonathanwalkerscripts/OppChasers#122]

**Comment by therealjustinmiller79** [comment]
Create an acceptance test-style checklist with the following: 

(Title) Bug Hunt

**On websites/applications of your choosing, see if you can find bugs in the following categories:**

- [ ] Image not appearing correctly
- [x] Link broken/goes to wrong page
- [x] Color Contrast not ADA Compliant
- [x] Missing Alt Text 
- [x] Present Alt Text (Show me a screenshot from Inspect!)
- [ ] Any Bug with Data Fields
- [ ] Design Flaw or Typo
- [x] Crash or Error Message
- [ ] Google Lighthouse ADA Score of exactly 80
- [ ] Any bug on CEO Works or a company you've worked with

Bonus: Use the inspect tool to change the headline of a major news story. Post a screenshot in the comments section.

**Comment by therealjustinmiller79** [comment]
multiple error messages!!

![image](https://user-images.githubusercontent.com/79952938/118205234-b8573280-b414-11eb-8d76-3b1c005e118c.png)

**Comment by therealjustinmiller79** [comment]
Multiple error messages on verizon .com

**Comment by therealjustinmiller79** [comment]
Link goes to wrong page (see issue #138)

**Comment by therealjustinmiller79** [comment]
color contrast not ada compliant 
![image](https://user-images.githubusercontent.com/79952938/118583107-5adf2080-b749-11eb-9077-e29f0cb8cf86.png)

**Comment by therealjustinmiller79** [comment]
missing alt text
![image](https://user-images.githubusercontent.com/79952938/118583966-c4136380-b74a-11eb-8065-61b0fba903a8.png)
